openfasoc/generators/temp-sense-gen/flow/scripts/detail_place.tcl

```diff
@@ -18,6 +18,23 @@ if {![info exists standalone] || $standalone} {
 } else {
   puts "Starting detailed placement"
 }
+
+# Note: Ali B Hammoud 8/11/22
+# template procedure which will place cells in the large voltage domain off east
+# with name "cell_name" semi-stacked starting from row "row_num" (0 indexed)
+# No error checking is used, so you must ensure the target row and block object are correct
+proc customPlace_east {block_object cell_name row_num} {
+	set target_row [lindex [$block_object getRows] $row_num]
+	set y_initial [expr {[lindex [$target_row getOrigin] 1] / 1000.0}]
+	set row_ydim [expr {[[$target_row getSite] getHeight] / 1000.0}]
+	foreach inst [$block_object getInsts] {
+		if {[[$inst getMaster] getName] == $cell_name} {
+			place_cell -cell $cell_name -inst_name [$inst getName] -origin [list 82.8 $y_initial] -orient R0 -status PLACED
```

Comment:
I think the `-status PLACED` has to be changed to `-status FIRM`, because otherwise the cells will be moved when `detailed_placement` is run